### PR TITLE
Fixed QUnit tests -  api select and content-editable api-actions (part of #471)

### PR DESCRIPTION
**Pull request by georgiy-abbasov**
\cc @AlexanderMoskovkin, @helen-dikareva

### CODE REVIEW

**Comment by georgiy-abbasov** [comment]
@testcafe-build-bot \retest

File: test/client/before-test.js [hunk 1]
```diff
@@ -77,4 +77,36 @@
     window.getCrossDomainPageUrl = function (filePath) {
         return window.QUnitGlobals.crossDomainHostname + window.QUnitGlobals.getResourceUrl(filePath);
     };
+
+    // NOTE: this method will override default window.scroll method and document.scrollTop
+    // property with our custom method and accessors with next behavior: document top
+    // position, which has been setted via runner API during test execution, is saved in
+    // state and will be applied when unexpected native scroll appears.
+    window.performScrollHack = function () {
```
Comment (AlexanderMoskovkin) [inline test/client/before-test.js]: I suppose it should be `window.preventNativeScrolling`

**Comment by helen-dikareva** [comment]
@VasilyStrelyaev

File: test/client/before-test.js [hunk 2]
```diff
@@ -77,4 +77,36 @@
     window.getCrossDomainPageUrl = function (filePath) {
         return window.QUnitGlobals.crossDomainHostname + window.QUnitGlobals.getResourceUrl(filePath);
     };
+
+    // NOTE: this method will override default window.scroll method and document.scrollTop
+    // property with our custom method and accessors with next behavior: document top
+    // position, which has been setted via runner API during test execution, is saved in
+    // state and will be applied when unexpected native scroll appears.
```
Comment (AlexanderMoskovkin) [inline test/client/before-test.js]: I think this comment contains some redundant information. Let's simplify it

Comment (AlexanderMoskovkin) [inline test/client/before-test.js]: Something like this:
```js
HACK: IOS browser performs unexpected scrolling in some conditions (<here is the link to issue>). With this hack we allow to set scroll only by script and prevent native browser scrolling
```

File: test/client/fixtures/runner/api/select-test/index-test.js [hunk 3]
```diff
@@ -232,6 +232,11 @@ $(document).ready(function () {
     if (browserUtils.isIE9)
         $(window.top.document).find('body').css('marginTop', '0px');
 
+    // NOTE: https://github.com/DevExpress/testcafe/issues/471
```
Comment (AlexanderMoskovkin) [inline test/client/fixtures/runner/api/select-test/index-test.js]: we can remove comment here because we comment it with this hack function definition

**Comment by AlexanderMoskovkin** [comment]
also, we should run the tests for 4-5 times to ensure we've fixed the problem

**Comment by AlexanderMoskovkin** [comment]
You've created a feature-branch in the main repo, not in your fork. As a result we have some unexpected testing behavior. So, I close the PR, please create it again from your fork

**Pull request closed without merging**